ui/styles/src/base/_admin.scss

```diff
@@ -486,9 +486,9 @@ div.pods-manage-field .pods-field-option-group .pods-field-option-group-label {
 /**
  * Pods DFV
  */
-.pods-manage-field .pods-dfv-field .pods-pick-values > ul {
-    border: 0;
-}
+// .pods-manage-field .pods-dfv-field .pods-pick-values > ul {
```

Comment:
We have a history in git, best to just remove lines unless there's another compelling reason here

Reply:
Affirmative. Shaking off the rust :)